CHANGELOG.rst

```diff
@@ -1,6 +1,20 @@
 TCP/IP Library Change Log
 =========================
 
```

Comment:
We're going to need to bump the version number in the module_build_info file and detail it here before this pull request is merged.